src/store.js

```diff
@@ -1,14 +1,14 @@
 var throttle = require('./lib/throttle');
 var toDenseArray = require('./lib/to_dense_array');
-var SimpleSet = require('./lib/simple_set');
+var StringSet = require('./lib/string_set');
 var render = require('./render');
 
 var Store = module.exports = function(ctx) {
   this._features = {};
-  this._featureIds = new SimpleSet();
-  this._selectedFeatureIds = new SimpleSet();
-  this._changedFeatureIds = new SimpleSet();
-  this._deletedFeaturesToEmit = new SimpleSet();
+  this._featureIds = new StringSet();
+  this._selectedFeatureIds = new StringSet();
+  this._changedFeatureIds = [];
```

Comment:
Not 100% confident about this so interested in your opinion ... I changed that to an array because it was only ever being added to and then used as array. I figured that it might save a little time on each render to leave it as an array, as opposed to needing to convert from an the set's object to an array. However, this list is usually relatively small, so I don't know if that matters or not. What do you think?